<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewFile</class>
 <widget class="QDialog" name="NewFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter the size of your image(1-64)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sizeInput">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="smallerNumber">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Please enter a valid number</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
